addons/medical_damage/ACE_Medical_Injuries.hpp

```diff
@@ -115,6 +119,10 @@ class ACE_Medical_Injuries {
             thresholds[] = {{0.1, 1}};
             selectionSpecific = 1;
         };
+        //No related wounds as drowning should not cause wounds/bleeding. Can be extended for internal injuries if they are added.
```

Comment:
Should this be left out until (if) airway management gets implemented?

Reply:
For now it is mainly there to prevent the wounds getting fired with Unkown damage type as that will cause wounds that make no sense for drowning. This was no issue in the current branch as the detection for a non specified damageType was not working correctly (the conversion to Unkown did not fully take place). This PR fixes that so it suddenly does become an issue.